Makefile

```diff
@@ -64,13 +64,23 @@ else
    # export GOOS ?= windows
 endif
 
-# Another person's PR is adding the debug support, so this is in prep for that
+#-----------------------------------------------------------------------------
+# Output control
+#-----------------------------------------------------------------------------
+VERBOSE ?= 0
+V ?= $(or $(VERBOSE),0)
+Q = $(if $(filter 1,$V),,@)
```

Comment:
using so short variables V,Q,H in a large makefile seems like it could yield troubles
also it's a bit cryptic what it does, add comments?
(I realize you are just moving those preexisting but still I only notice them now)

Reply:
They're the one bit of cryptic make that I can't take the blamed for. :)

The benefit of being short will decrease as more targets learn to share the same recipe/rule (e.g., all the docker images are down to 2 rules [DOCKER_GENERIC_RULE / DOCKER_SPECIFIC_RULE] and renaming a few "Dockerfile"s would get this down to 1).

Reply:
I took a look at what the Q variable is there for, which is to echo the commands being executed. The 'H' variable put a header (equivalent to =>) for informational output. 

If you guys think that it's a good idea to rename them as part of this patch, I can do that. Or I can simply add comments. let me know either way.

Reply:
I don't have any additional expectations beyond what has already been done.

Reply:
I removed the variable V since it's used only once and not exported, and added comments about the variables Q and H. I didn't rename them to minimize the change in this PR.

Reply:
in my opinion all commands except echo should be shown in makefile (ie one should never use @command unless it's @echo ...) but I guess it's cool to have a Quiet mode
Just comment would be nice